src/components/Game/Game.js

```diff
@@ -1,20 +1,49 @@
-import React, { useContext } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
-import Button from "../Button";
 import "./Game.css";
-import { GameContext } from "../../contexts/gameContext";
+import Avatar from "../Avatar";
+import Button from "../Button";
+import PlayerCard from "../PlayerCard/PlayerCard";
 import ScoreInput from "../ScoreInput/ScoreInput";
+import Status from "../Status";
 import Swords from "../../images/swords.svg";
+import { GameContext } from "../../contexts/gameContext";
 import { Player } from "../../utils/player";
 
 const Game = props => {
   const game = useContext(GameContext);
   const [players, setPlayers] = game.usePlayers;
+  const [lowestPlayerLevel, setLowestPlayerLevel] = useState();
+  const [highestPlayerLevel, setHightestPlayerLevel] = useState();
+  const [isToggleSex, setIsToggleSex] = useState(false);
+
+  console.log(players);
+  useEffect(() => {
+    const levels = players.map(player => player.level);
+    const max = Math.max(...levels);
+    const min = Math.min(...levels);
+    const minLevelLength = levels.filter(level => min === level).length;
+    setLowestPlayerLevel(minLevelLength === 1 ? min : null);
+    setHightestPlayerLevel(max);
+  }, [players]);
+
+  const updatePlayerLevel = (newLevel, playerIndex) => {
```

Comment:
These update handlers can probably be combined into a single `updatePlayer` handler (this could also be used to handle the sex change! )

Reply:
I was able to factor this for bonus and level, but how I wrote the sex toggle, it is not working